druid/src/widget/svg.rs

```diff
@@ -150,20 +156,24 @@ impl SvgData {
                                 x,
                                 y,
                             } => {
-                                let x1 = (x1 * scale) + offset.x;
-                                let y1 = (y1 * scale) + offset.y;
-                                let x2 = (x2 * scale) + offset.x;
-                                let y2 = (y2 * scale) + offset.y;
-                                let x = (x * scale) + offset.x;
-                                let y = (y * scale) + offset.y;
-
                                 path.curve_to((x1, y1), (x2, y2), (x, y));
                             }
                             usvg::PathSegment::ClosePath => {
                                 path.close_path();
                             }
                         }
                     }
+
+                    path.apply_affine(Affine::new([
```

Comment:
annoying that this conversion isn't easier to do, especially since `usvg` _does_ have a dependency on kurbo..

Reply:
@razrfalcon do you have any interest in using kurbo for more of the basic geom stuff in usvg?  No worries either way, but if you're already bringing it in for other stuff it might let you delete some code.

Reply:
>I do assume there's a good reason you want your own impls

Yes, in my case, my geometry primitives are immutable and always valid. Which simplify things a lot. I plan to move them into a separate crate eventually.

Also, I will move back to `lyon` when `raqote` backend became default, since it already used it.

Reply:
sounds good, thanks!